centos-7.3-x86_64-openstack.json

```diff
@@ -0,0 +1,50 @@
+{
+  "builders": [
+    {
+      "boot_command": [
+        "<tab> text ks=http://{{ .HTTPIP }}:{{ .HTTPPort }}/centos-7.3/ks-openstack.cfg<enter><wait>"
+      ],
+      "accelerator": "kvm",
+      "boot_wait": "10s",
+      "disk_size": 2048,
```

Comment:
The disk size is smaller than the one in https://github.com/osuosl/packer-templates/blob/master/centos-7.3-ppc64le-openstack.json#L13 so I don't think that's the root cause of this inflated image.